catalog/app/components/Collaborators/Table.tsx

```diff
@@ -0,0 +1,94 @@
+import * as R from 'ramda'
+import * as React from 'react'
+import * as M from '@material-ui/core'
+
+import * as Model from 'model'
+
+type SortKey = 'email' | 'username' | 'permissionLevel'
+type Sort = Record<SortKey, boolean> // TODO: only one key at a time is allowed
+
+function getSortProperty(key: SortKey) {
+  switch (key) {
+    case 'email':
+      return R.view(R.lensPath(['collaborator', 'email']))
+    case 'username':
+      return R.view(R.lensPath(['collaborator', 'username']))
```

Comment:
is this different from `R.path()`?

Reply:
Should be the same, but `R.path` didn't work here. I re-check, maybe, it was some other mistake

Reply:
maybe related to null/undefined handling